### PR TITLE
McKenna.md

### PR DESCRIPTION
**Pull request by mckennaheasley**

### CODE REVIEW

File: McKenna.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Here are my two truths and a lie
+
+1. I lived in OR
+2. I lived in MA
+3. I lived in CA
```
Comment (JoshuaToenyes) [inline McKenna.md:5]: 😂 I'll flip a coin and say that `MA` is the lie.

**Comment by kdp0123** [comment]
I'm going with CA is the lie!

File: McKenna.md (hunk 1 above)
Comment (wardelltiffany) [inline McKenna.md:5]: I'm guessing living in California is the lie.

Comment (Steingraber) [reply McKenna.md:5]: I am going to go with MA as well.

**Review by pcook21 — COMMENTED** [review]
#2 is the lie

**Comment by ruggedstickman** [comment]
nobody has 6 kids :)

**Pull request merged**